Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ConnectionPage.tsx

```diff
@@ -225,7 +217,7 @@ export default function ConnectionPage(props: IProps) {
                     <label>Select Connecting Asset</label>
                     <select value={selectedAssetKey} className="form-control" onChange={(evt) => { setSelectedAssetKey((evt.target.value) as string); }}>
                         {
-                            props.AllAssets.filter(asset => asset.AssetKey != props.CurrentAsset.AssetKey).map((asset, index) => <option key={index} value={asset.AssetKey} >{asset.AssetKey}</option>)
+                            props.AllAssets.filter(asset => asset.AssetKey != props.CurrentAsset.AssetKey).map((asset, index) => <option key={index} value={asset.AssetKey} >{asset.AssetName}</option>)
```

Comment:
should be `asset.AssetName (asset.AssetKey)`

Reply:
That's backwards, it would cause the value internally to be (asset.AssetName) and the label to be (asset.AssetKey)

Reply:
no I mean just the Text should be that. It should show to the user as `Bus A (B1)`

Reply:
Ahh I see, will do.